homeassistant/components/rachio/switch.py

```diff
@@ -276,3 +288,98 @@ def _handle_update(self, *args, **kwargs) -> None:
         """Unsubscribe from updates."""
         if self._undo_dispatcher:
             self._undo_dispatcher()
+
+
+class RachioSchedule(RachioSwitch):
+    """Representation of one fixed schedule on the Rachio Iro."""
+
+    def __init__(self, person, controller, data, current_schedule):
+        """Initialize a new Rachio Schedule."""
+        self._id = data[KEY_ID]
+        self._schedule_name = data[KEY_NAME]
+        self._duration = data[KEY_DURATION]
+        self._schedule_enabled = data[KEY_ENABLED]
+        self._summary = data[KEY_SUMMARY]
+        self._current_schedule = current_schedule
+        super().__init__(controller, poll=False)
+        self._state = self.schedule_id == self._current_schedule.get(KEY_SCHEDULE_ID)
+        self._undo_dispatcher = None
+
+    @property
+    def schedule_id(self) -> str:
+        """How the Rachio API refers to the schedule."""
+        return self._id
+
+    @property
+    def name(self) -> str:
+        """Return the friendly name of the schedule."""
+        return self._schedule_name
```

Comment:
```suggestion
        return f"{self._schedule_name} Schedule"
```

It was confusing to tell these apart from zones

Reply:
Good point. I named all mine "__ Schedule" for testing..